sdk/cosmos/azure-spring-data-cosmos-core/src/samples/java/com/azure/cosmos/PageableRepositoryCodeSnippet.java

```diff
@@ -1,39 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are we removing this code snippet ?
Is it not getting used anywhere ?

Reply:
Yes, it not used. I deleted it and integration test passed.

Reply:
It used in readme, I'll recover it.